### DISCUSSION TITLE
@apply with an imported preset errors

### DISCUSSION THREAD

**Discussion by tessamerrill** [tailwindlabs/tailwindcss#8947] [Help]
Hi there!

I'm facing an issue where when using `@apply` within a `.scss` file with an imported preset class errors.

Example:
```scss
 &.class {
                    @apply bg-gray-800 bg-opacity-5 dark:bg-new-blue-300 dark:bg-opacity-12;
                }
```
TW config:
```js
const config = {
    presets: [require('../../tailwind.base.config.js')],
    // rest of config
}

// base.config

module.exports = {
    colors: {
        'new-blue-300': '#7DD3FC',
    }
};
```
(code is minified from existing codebase)

Any help is appreciated, thanks!

**Comment by wongjn** [comment]
The error should tell you the exact cause of it. It seems you are using v2 of Tailwind right? Assuming the rest of your redacted config is unrelated, it might be because `bg-opacity-12` class does not exist.
